src/components/AddTodos.tsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { Form, Formik, useFormik, Field, ErrorMessage } from "formik";
+import * as yup from "yup";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { v4 as uuidv4, validate } from "uuid";
+
+const AddTodos = () => {
+  const navigate = useNavigate();
+  const unique_id = uuidv4;
+
+  const validationSchema = yup.object({
+    title: yup.string().required("Title is required"),
+    description: yup.string().required("Description is required"),
+    assignee: yup.string().required("Assignee is required"),
+    date: yup.date()
+      .min(new Date(), "due date cannot be less than current data")
+      .required("Due date is required"),
+  });
+
+  const formik = useFormik({
+    initialValues: {
```

Comment:
Declare initialValues above the component and use here

Reply:
Done